packages/com.sncommunity.markdown-monospace/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@standardnotes/markdown-monospace",
+  "version": "1.0.0",
+  "main": "dist/dist.css",
+  "author": "DanielNetoP",
+  "description": "Ability to toggle monospace font for Markdown, Minimal Markdown, and Markdown Visual plugins.",
```

Comment:
This is not for markdown visual, it works on these 3 only
- Basic Markdown
- Minimal Markdown
- Markdown

cc: @moughxyz